README.md

```diff
@@ -1,172 +1,54 @@
+* The WhatWG loader is not finalized and is unstable and liable to change. Consequently, this polyfill is also liable to change. This is unlikely to be implemented in browsers until stable.
+
+### WhatWG loader status June 2016
+* Defines a `Loader` class in the (new to ES2015) `Reflect` global object, with `import`, `resolve` and `load` prototype methods.
+* The `Loader` class also includes a module `Registry`, which is a `Symbol.iterator`.
+* Introduces a `System` global object, with an instance of `Reflect.Loader` as the `loader` property; this is the default browser loader. In addition to the `Loader` methods, this has `resolve`, `fetch`, `translate` and `instantiate` prototype methods, of which the last two are by default no-ops.
+* Defines a `Module` constructor in the `Reflect` global for dynamic construction of modules from within script or module code.
+
+### This polyfill
+* Implements `Reflect.Loader`, providing the ability for developers to create custom loader instances.
+* Implements the module registry
+* Implements `Reflect.Module`
+* Implements `System.loader`, which means that `System.loader.import` can be used to dynamically load ES6 modules.
+* Until the parsing of `import` and `export` statements are natively supported, supports both [Traceur](https://github.com/google/traceur-compiler) and [Babel](http://babeljs.io/) for transpiling these statements; this requires loading a large parser, so is not suitable for production use in the browser, though can of course be used in NodeJS without problem.
+* Fully supports ES6 circular references and live bindings.
+* Targets modern browsers, all of which now support the great bulk of ES6.
+
+### Differences with versions prior to 0.50
+* As the target browsers support ES6, no polyfills are included; applications wanting to support older browsers, such as IE, need to include polyfills for `Map`, `Symbol.iterator`, and `Promise`, as well as `URL()`; https://cdn.polyfill.io provides a simple way to do this.
+* As only `import` and `export` are transpiled, those wanting to support older browsers should also pre-transpile any ES6 code into ES5.
+* What was previously in the `System` object, for example `System.import`, is now in `System.loader` (applications can of course supply an alias to ease the transition).
+* The previous `normalize` and `locate` methods have now been combined into the `resolve` method.
+* The `Module` constructor has been moved to the `Reflect` object.
+* Although the WhatWG loader will eventually define a `site` object, which will probably include some kind of module name/URL mapping, this is not currently defined and so is not included in the current polyfill; this means there is no longer any `baseURL` or `paths` support. For compatibility with script modules, the default loader only supports relative URLs `/`, `./` and `../` in `import` statements (this is relative to the URL of the importing module).
```

Comment:
An important point here though is that this is a hookable loader - users can make their own loaders with it. That is perhaps the strongest use case for using this loader polyfill for more than experimentation - to be able to have a loader in use that is based on a standard.